model/client/transactions/submit_request.go

```diff
@@ -8,3 +8,7 @@ type SubmitRequest struct {
 func (*SubmitRequest) Method() string {
 	return "submit"
 }
+
+func (*SubmitRequest) Validate() error {
+	return nil
```

Comment:
Need to check tx_blob as it's required

Reply:
Ohhhh I've just thought - are these going to be validated differently? We need to build out our submit methods but these will build out the tx_blob anyway, so won't need to validate on it?

Reply:
We can have this request constructed by a helper method that would validate the transaction during construction
This request should probably still validate that the blob is at least set